libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -142,7 +142,7 @@ export const DealTicket = ({
   );
   const { data: activeOrders } = useDataProvider({
     dataProvider: activeOrdersProvider,
-    variables: { partyId: pubKey || '' },
+    variables: { partyId: pubKey || '', marketId: market.id },
```

Comment:
Good find 👍. Something to consider though.. It may be better to not change the variable and instead filter the resulting orders (perhaps use a derived data provider, or a wrapping hook), thus avoiding keeping two stores in sync at the same time. By changing the variable I think we may end up with duplicated orders in two different stores?

@bglownia would love some input here

Reply:
Looks like activeOrdersProvider is already handling the filtering (after the query resolves) internally 👍